JavaErickHernandez/Day3/HashSets.java

```diff
@@ -0,0 +1,55 @@
+package bootcamp;
+import java.util.Scanner;
+import java.util.Set;
+import java.util.HashSet;
+
+public class HashSets {
+	static String numberOfInputs = "Enter the number of pairs: ";
```

Comment:
What is the reason for not putting these variables as "final"? if you put "final" you avoid to any other person can change this variables in other par of the code.